WordPress/Classes/System/WordPressAppDelegate+openURL.swift

```diff
@@ -1,74 +1,58 @@
 import WordPressAuthenticator
 
-/// Custom pattern matcher used for our URL scheme
-private func ~=(pattern: String, value: URL) -> Bool {
-    return value.absoluteString.contains(pattern)
-}
-
-private extension Array where Element == URLQueryItem {
-    func value(of key: String) -> String? {
-        return self.first(where: { $0.name == key })?.value
-    }
-    func intValue(of key: String) -> Int? {
-        guard let value = value(of: key) else {
-            return nil
-        }
-        return Int(value)
-    }
-}
-
-private extension URL {
-    func queryParams() -> [URLQueryItem]? {
-        guard let components = URLComponents(url: self, resolvingAgainstBaseURL: false),
-            let queryItems = components.queryItems,
-            queryItems.count > 0 else {
-                return nil
-        }
-        return queryItems
-    }
-}
-
 @objc extension WordPressAppDelegate {
     internal func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey: Any] = [:]) -> Bool {
         DDLogInfo("Application launched with URL: \(url)")
-        var returnValue = false
 
-        // 1. check if hockey can open this URL
-        var hockeyOptions: [String: Any] = [:]
-        for (key, value) in options {
-            hockeyOptions[key.rawValue] = value
-        }
-        if hockey.handleOpen(url, options: hockeyOptions) {
-            returnValue = true
+        guard !handleHockey(url: url, options: options) else {
+            return true
         }
 
-        // 2. check if this is a Google login URL
-        if WordPressAuthenticator.shared.handleGoogleAuthUrl(url,
-                                                             sourceApplication: options[UIApplication.OpenURLOptionsKey.sourceApplication] as? String,
-                                                             annotation: options[UIApplication.OpenURLOptionsKey.annotation]) {
-            returnValue = true
+        guard !handleGoogleAuth(url: url, options: options) else {
+            return true
         }
 
         // 3. let's see if it's our wpcom scheme
         guard url.scheme == WPComScheme else {
-            return returnValue
+            return false
         }
 
         // this works with our custom ~=
         switch url {
         case "magic-login":
-            returnValue = handleMagicLogin(url: url)
+            return handleMagicLogin(url: url)
         case "viewpost":
-            returnValue = handleViewPost(url: url)
+            return handleViewPost(url: url)
         case "viewstats":
-            returnValue = handleViewStats(url: url)
+            return handleViewStats(url: url)
         case "debugging":
             handleDebugging(url: url)
```

Comment:
I think let's update `handleDebugging` so it returns a Bool – then it matches all the other `handle` methods in here?